### DISCUSSION TITLE
Strong Model Match. UID shifting for one passphrase on some TX and RX

### DISCUSSION THREAD

**Discussion by r1000ru** [ExpressLRS/ExpressLRS#3057] [Ideas]
It is currently not possible to use two or more transmitters with the same password phrase with multiple receivers at the same time.
This is relevant when I want to ride cars with children or friends. 

I propose implementing a "strict modelmatch" mode, when using which, the UID will be shifted to the left by one byte and appended with the rxID value, as variant. Thus, having selected the desired model on each transmitter, the connection will be established specifically with it, without interfering with another transmitter.

I understand that such a mode is not in demand for fcopters and airplanes, but for ground vehicles - it is very relevant.

**Comment by pkendall64** [comment]
Or you could use the volatile binding method so that the receiver always enters binding mode when first powered. Then you just bind it using the [Bind] button in the Lua script, then each transmitter can have it's own binding phrase.
See https://www.expresslrs.org/quick-start/binding/?h=#updated-binding-procedure-since-expresslrs-340

**Comment by r1000ru** [comment]
Definitely a good solution, but requires a bind after each battery change

**Comment by mm3x** [comment]
Are you referring to situation with two different Model-match values or you have a reason to disable Model-match / share model number?
